di-ipv-evcs-stub/lambdas/src/domain/enums/vcState.ts

```diff
@@ -0,0 +1,26 @@
+enum VcState {
+  CURRENT = "CURRENT",
+  PENDING = "PENDING",
+  VERIFICATION = "VERIFICATION",
+  ABANDONED = "ABANDONED",
+  PENDING_RETURN = "PENDING_RETURN",
+  HISTORIC = "HISTORIC",
+  VERIFICATION_ARCHIVED = "VERIFICATION_ARCHIVED",
+}
+
+export const CreateVcStates = {
+  [VcState.CURRENT]: VcState.CURRENT,
+  [VcState.PENDING]: VcState.PENDING,
+  [VcState.PENDING_RETURN]: VcState.PENDING_RETURN,
+  [VcState.VERIFICATION]: VcState.VERIFICATION,
+} as const;
+
+export const UpdateVcStates = {
+  [VcState.CURRENT]: VcState.CURRENT,
+  [VcState.ABANDONED]: VcState.ABANDONED,
+  [VcState.PENDING_RETURN]: VcState.PENDING_RETURN,
+  [VcState.HISTORIC]: VcState.HISTORIC,
+  [VcState.VERIFICATION_ARCHIVED]: VcState.VERIFICATION_ARCHIVED,
+} as const;
+
+export default VcState;
```

Comment:
Just a personal preference, but I've always stayed away from default exports in a file where you are also exporting other functions, just to avoid confusion (either that or have them in separate files and default export them)

Reply:
done